catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -1,57 +1,120 @@
-import { FunctionComponent, useState } from 'react';
+import { FunctionComponent, useEffect, useState, useCallback } from 'react';
+import Gallery from 'react-photo-gallery';
+import styled from 'styled-components';
+
+import { Image, CatalogueResultsList } from '@weco/common/model/catalogue';
+
 import ExpandedImage from '../ExpandedImage/ExpandedImage';
 import ImageCard from '../ImageCard/ImageCard';
-import { Image, CatalogueResultsList } from '@weco/common/model/catalogue';
 import Modal from '@weco/common/views/components/Modal/Modal';
+import { convertImageUri } from '@weco/common/utils/convert-image-uri';
 
 type Props = {
   images: CatalogueResultsList<Image>;
 };
 
+const imageMargin = 16;
+
+const GalleryContainer = styled.div`
+  margin: 0 -${imageMargin / 2}px;
+
+  // This div is rendered by Gallery so we're targetting it on mobile only to ensure images are aligned on both sides
+  .react-photo-gallery--gallery > div {
+    justify-content: space-around;
+  }
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    margin: 0 -${imageMargin}px;
+
+    .react-photo-gallery--gallery > div {
+      justify-content: flex-start;
+    }
+  }
+`;
+const ImageContainer = styled.li`
+  margin: 0 ${imageMargin ? imageMargin / 2 : 0}px
+    ${imageMargin ? imageMargin / 2 : 0}px;
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    margin: 0 ${imageMargin}px ${imageMargin}px;
+  }
+`;
+
 const ImageEndpointSearchResults: FunctionComponent<Props> = ({
   images,
 }: Props) => {
   const [expandedImage, setExpandedImage] = useState<Image | undefined>();
+  const [data, setData] = useState([]) as any;
+
   // In the case that the modal changes the expanded image to
   // be one that isn't on this results page, this index will be -1
   const expandedImagePosition = images.results.findIndex(
     img => expandedImage && img.id === expandedImage.id
   );
   const [isActive, setIsActive] = useState(false);
 
+  useEffect(() => {
+    if (data.length) setData([]);
+
+    images.results.map(async image => {
+      setData(prev => {
+        return [
+          ...prev,
+          {
+            ...image,
+            src: convertImageUri(image.locations[0].url, 300),
+            width: (image.aspectRatio || 1) * 100 + imageMargin,
+            height: 100,
+          },
+        ];
+      });
+    });
+  }, [images]);
+
+  const imageRenderer = useCallback(galleryImage => {
+    return (
+      <ImageContainer key={galleryImage.key} role="listitem">
+        <ImageCard
+          id={galleryImage.photo.id}
+          workId={galleryImage.photo.workId}
+          image={{
+            contentUrl: galleryImage.photo.src,
+            width: galleryImage.photo.width - imageMargin * 2,
+            height: galleryImage.photo.height,
+            alt: '',
```

Comment:
Why no alt text?

Edit: ah, I see, this is inherited from a previous version.

Reply:
I wondered about that, too. I'm not sure the image title is the best alt text either but I could use it in the meantime. Just conscious that it's sometimes extra long and might not be useful to users, but better than nothing?

Reply:
Ah, so some useful context to know is that we deliberately omit alt text on promo images for Prismic. See the mixin here: https://github.com/wellcomecollection/wellcomecollection.org/blob/main/content/webapp/services/prismic/transformers/images.ts#L58-L71

They're not quite the same, because Prismic content has other information for a screen reader to latch on to.

Let's leave it as-is for now, and make a note to ask about it at the next accessibility audit.

Reply:
Ah good to know, but agree it's not the same, there's zero text and since they are literally the goal of the page and not decorative images, reckon an alt stays useful. When do these audits occur?

Reply:
Another good question! I say that, I don't actually know. 🤔